src/app.scss

```diff
@@ -0,0 +1 @@
+@import './components/TutorialHeader/tutorial-header.scss';
```

Comment:
You're missing the landing page and repo page imports. See https://www.carbondesignsystem.com/tutorial/react/step-1#set-up-content-sass